identifiers/country-us/state-ky-local_gov.csv

```diff
@@ -288,6 +273,18 @@ ocd-division/country:us/state:ky/county:letcher/council_district:5,Letcher Count
 ocd-division/country:us/state:ky/county:lewis/council_district:1,Lewis County District 1
 ocd-division/country:us/state:ky/county:lewis/council_district:2,Lewis County District 2
 ocd-division/country:us/state:ky/county:lewis/council_district:3,Lewis County District 3
+ocd-division/country:us/state:ky/county:fayette/council_district:1,Lexington-Fayette Urban County Government district 1
```

Comment:
changes pushed for KY

Reply:
It looks like `ocd-division/country:us/state:ky/county:fayette/council_district:1` is still after `ocd-division/country:us/state:ky/county:lewis/council_district:3` instead of after `ocd-division/country:us/state:ky/county:estill/council_district:3`.

Reply:
Sorry about that, must've sorted the name not the ocd-id. Should be in the right order now